docs/source/intervention_models/iv_iron_postpartum/index.rst

```diff
@@ -65,13 +45,20 @@ Assume no baseline coverage in low and middle income countries.
 Vivarium Modeling Strategy
 --------------------------
 
-.. todo::
+We will model the following eligibility for the postpartum IV iron intervention:
 
-  Add an overview of the Vivarium modeling section.
+#. Pregnant women at birth (i.e. the transition from the pregnant to postpartum states) according to the :ref:`pregnancy model document <other_models_pregnancy>`
+#. Hemoglobin level of less than 100 grams per liter according to the :ref:`hemoglobin model document <2019_hemoglobin_model>` *after* the application of the pregnancy adjustment factor and any other factors that affect hemoglobin level such as pre-birth intervention coverage or maternal hemorrhage shift.
 
-.. todo::
+.. note::
 
-  Fill out the following table with all of the affected measures that have vivarium modeling strategies documented
+  Women who receive IV iron should continue to be covered by the :ref:`maternal supplementation intervention <maternal_supplementation_intervention>` 
+
+  Coverage by :ref:`antenatal IV iron <intervention_iv_iron_postpartum>` is not an exclusion criteria for coverage by postpartum IV iron
+
+  A potential future eligibility criterion for this intervention may be in-facility delivery and/or skilled birth attendance if we include them as modeled parameters
+
+We assume that IV iron intervention coverage takes place at the start of pregnancy (i.e. upon transition into the pregnant state).
```

Comment:
post-partum iron takes place at birth ??

Reply:
Oops, yes, copy/paste error thank you for catching this.